plugins/ftdidmx/FtdiWidget.cpp

```diff
@@ -192,7 +192,8 @@ void FtdiWidget::Widgets(vector<FtdiWidgetInfo> *widgets) {
         if (std::string::npos != v.find("FTDI") ||
             std::string::npos != v.find("KMTRONIC") ||
             std::string::npos != v.find("KWMATIK") ||
-            std::string::npos != v.find("WWW.SOH.CZ")) {
+            std::string::npos != v.find("WWW.SOH.CZ") ||
+            std::string::npos != v.find("NORTLE")) {
```

Comment:
Can you sort the order of the manufacturers please.

Reply:
what do you mean by sort manufacturers

Reply:
As in WWW.SOH.CZ > NORTLE, the string comparisons.